src/main/resources/assets/createframed/lang/ru-ru.json

```
@@ -0,0 +1,217 @@
+  "block.createframed.blue_stained_framed_glass_door": "Синяя витражная обрамлённая стеклянная дверь",
+  "block.createframed.light_blue_stained_framed_glass_door": "Голубая витражная обрамлённая стеклянная дверь",
+  "block.createframed.cyan_stained_framed_glass_door": "Циановая витражная обрамлённая стеклянная дверь",
+  "block.createframed.purple_stained_framed_glass_door": "Фиолетовая витражная обрамлённая стеклянная дверь",
+  "block.createframed.magenta_stained_framed_glass_door": "Пурпурная витражная обрамлённая стеклянная дверь",
+  "block.createframed.pink_stained_framed_glass_door": "Розовая витражная обрамлённая стеклянная дверь",
+  "block.createframed.black_stained_framed_glass_door": "Чёрная витражная обрамлённая стеклянная дверь",
+  "block.createframed.gray_stained_framed_glass_door": "Серая витражная обрамлённая стеклянная дверь",
+  "block.createframed.light_gray_stained_framed_glass_door": "Светло-серая витражная обрамлённая стеклянная дверь",
+  "block.createframed.white_stained_framed_glass_door": "Белая витражная обрамлённая стеклянная дверь",
+  "block.createframed.brown_stained_framed_glass_door": "Коричневая витражная обрамлённая стеклянная дверь",
+  "block.createframed.tinted_framed_glass_door": "Тонированная обрамлённая стеклянная дверь",
+
+  "block.createframed.red_stained_framed_glass_trapdoor": "Красный витражный обрамлённый люк",
+  "block.createframed.orange_stained_framed_glass_trapdoor": "Оранжевый витражный обрамлённый люк",
+  "block.createframed.yellow_stained_framed_glass_trapdoor": "Жёлтый витражный обрамлённый люк",
+  "block.createframed.green_stained_framed_glass_trapdoor": "Зелёный витражный обрамлённый люк",
+  "block.createframed.lime_stained_framed_glass_trapdoor": "Лаймовый витражный обрамлённый люк",
+  "block.createframed.blue_stained_framed_glass_trapdoor": "Синий витражный обрамлённый люк",
+  "block.createframed.light_blue_stained_framed_glass_trapdoor": "Голубой витражный обрамлённый люк",
+  "block.createframed.cyan_stained_framed_glass_trapdoor": "Циановый витражный обрамлённый люк",
+  "block.createframed.purple_stained_framed_glass_trapdoor": "Фиолетовый витражный обрамлённый люк",
+  "block.createframed.magenta_stained_framed_glass_trapdoor": "Пурпурный витражный обрамлённый люк",
+  "block.createframed.pink_stained_framed_glass_trapdoor": "Розовый витражный обрамлённый люк",
+  "block.createframed.black_stained_framed_glass_trapdoor": "Чёрный витражный обрамлённый люк",
+  "block.createframed.gray_stained_framed_glass_trapdoor": "Серый витражный обрамлённый люк",
+  "block.createframed.light_gray_stained_framed_glass_trapdoor": "Светло-серый витражный обрамлённый люк",
+  "block.createframed.white_stained_framed_glass_trapdoor": "Белый витражный обрамлённый люк",
+  "block.createframed.brown_stained_framed_glass_trapdoor": "Коричневый витражный обрамлённый люк",
+  "block.createframed.tinted_framed_glass_trapdoor": "Тонированный обрамлённый люк",
+
+  // Alex's Caves
```

I don't know, maybe it's worth adding a dummy key:value here to appease the formatter.
```suggestion
  "//": "Alex's Caves",
```
I remember in some mod it broke the localization of all the lines below, but maybe Minecraft has fixed it since then.

---

I'll just remove the comment then. I'm not sure if it would prevent the rest of the localization file from being loaded, but the comment's not really important to keep.